src/include/daos/mem.h

```diff
@@ -987,13 +989,15 @@ struct umem_cache_chkpt_stats {
  * \param[in]	base_off	Offset of the umem cache base
  * \param[in]	base		Start address of the page cache
  * \param[in]	is_evictable_fn	Callback function to check if page is evictable
+ * \param[in]	pageload_fn	Callback called on page being loaded
  *
  * \return 0 on success
  */
 int
 umem_cache_alloc(struct umem_store *store, uint32_t page_sz, uint32_t md_pgs, uint32_t mem_pgs,
 		 uint32_t max_ne_pgs, uint32_t base_off, void *base,
-		 bool (*is_evictable_fn)(uint32_t pg_id));
+		 bool (*is_evictable_fn)(uint32_t pg_id),
```

Comment:
We need to pass arg to is_evictable_fn also. May be set the type of arg as struct umem_cache

Reply:
sure, I'll pass arg to this function too. Probably keeping it as "void *", since this function is declared in the "struct umem_cache".